### PR TITLE
Adds a load of new FBPs.

### PR DESCRIPTION
**Pull request by Belsima**
Sprites from @SubberTheFabulous and Aurora. 

image
image
image
image

Some of them have bugs with eyes displaying, but so do our current monitor heads, even the original Morpheus. I'll leave it as 'make your eyes fit the FBP's color I guess' since last time I touched human hair I broke FBP hair, so I really don't want to touch eyes.

### CODE REVIEW

**Comment by elgeonmb** [comment]
My official title is Executor.

**Comment by Spacemanspark** [comment]
Not sure if you'd consider this out of scope, but you should totally port the IPC/ FPB antenna from Paradise.

**Comment by Anewbe** [comment]
Seeing as I've literally never heard mention of any of those companies... Gonna have to take some time to review this.

**Comment by Belsima** [comment]
Alright. Elgeon just detailed all of them to me in PMs, so I threw it up.

**Comment by elgeonmb** [comment]
They're supposed to be knockoff corporation names, I don't really think it's strictly-speaking necessary to detail the backstory behind the 2561 version of bootleg playstations. We're kind of _out_ of corporations that might reasonably produce FBPs other than the PMC ones.

**Comment by Belsima** [comment]
Cyber solutions sprites changed to these: https://github.com/PolarisSS13/Polaris/pull/3833 given talks on the PR and discord.

File: code/datums/supplypacks/robotics.dm [hunk 1]
```diff
@@ -60,6 +60,14 @@
 	containername = "Robolimb blueprints (Morpheus)"
 	access = access_robotics
 
+/datum/supply_packs/robotics/robolimbs/cybersolutions
+	name = "Cyber Solutions robolimb blueprints"
+	contains = list(/obj/item/weapon/disk/limb/cybersolutions)
+	cost = 20
```
Comment (Anewbe) [inline code/datums/supplypacks/robotics.dm]: These prices are all going to need to be discussed.

File: code/datums/supplypacks/robotics.dm [hunk 2]
```diff
@@ -68,6 +76,14 @@
 	containername = "Robolimb blueprints (Xion)"
 	access = access_robotics
 
+/datum/supply_packs/robotics/robolimbs/hypnos
+	name = "Hypnos robolimb blueprints"
+	contains = list(/obj/item/weapon/disk/limb/hypnos)
+	cost = 15
```
Comment (Anewbe) [inline code/datums/supplypacks/robotics.dm]: These prices are all going to need to be discussed.

File: code/datums/supplypacks/robotics.dm [hunk 3]
```diff
@@ -84,6 +100,14 @@
 	containername = "Robolimb blueprints (Hephaestus)"
 	access = access_robotics
 
+/datum/supply_packs/robotics/robolimbs/athena
+	name = "Athena robolimb blueprints"
+	contains = list(/obj/item/weapon/disk/limb/athena)
+	cost = 25
```
Comment (Anewbe) [inline code/datums/supplypacks/robotics.dm]: These prices are all going to need to be discussed.

File: code/datums/supplypacks/robotics.dm [hunk 4]
```diff
@@ -92,6 +116,14 @@
 	containername = "Robolimb blueprints (Ward-Takahashi)"
 	access = access_robotics
 
+/datum/supply_packs/robotics/robolimbs/wasikeltabet
+	name = "Wasik/el-Tabet robolimb blueprints"
+	contains = list(/obj/item/weapon/disk/limb/wardtakahashi)
+	cost = 25
```
Comment (Anewbe) [inline code/datums/supplypacks/robotics.dm]: These prices are all going to need to be discussed.

File: code/datums/supplypacks/robotics.dm [hunk 5]
```diff
@@ -92,6 +116,14 @@
 	containername = "Robolimb blueprints (Ward-Takahashi)"
 	access = access_robotics
 
+/datum/supply_packs/robotics/robolimbs/wasikeltabet
+	name = "Wasik/el-Tabet robolimb blueprints"
+	contains = list(/obj/item/weapon/disk/limb/wardtakahashi)
```
Comment (Anewbe) [inline code/datums/supplypacks/robotics.dm]: Give this another look, please.

File: code/datums/supplypacks/robotics.dm [hunk 6]
```diff
@@ -92,6 +116,14 @@
...
+	cost = 25
+	containertype = /obj/structure/closet/crate/secure/science
+	containername = "Robolimb blueprints (Ward-Takahashi)"
```
Comment (Anewbe) [inline code/datums/supplypacks/robotics.dm]: Give this another look, please.

File: code/datums/supplypacks/robotics.dm [hunk 7]
```diff
@@ -108,6 +140,14 @@
 	containername = "Robolimb blueprints (Bishop)"
 	access = access_robotics
 
+/datum/supply_packs/robotics/robolimbs/rook
+	name = "Rook robolimb blueprints"
+	contains = list(/obj/item/weapon/disk/limb/rook)
+	cost = 55
```
Comment (Anewbe) [inline code/datums/supplypacks/robotics.dm]: These prices are all going to need to be discussed.

File: code/modules/organs/robolimbs.dm [hunk 8]
```diff
@@ -82,6 +82,25 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
...
+	icon = 'icons/mob/human_races/cyberlimbs/rook/rook_main.dmi'
+	unavailable_to_build = 1
+
+/datum/robolimb/cybersolutions
+	company = "Cyber Solutions"
+	desc = "This limb is grey and rough, with little in the way of aesthetic."
+	icon = 'icons/mob/human_races/cyberlimbs/cybersolutions/cybersolutions_main.dmi'
+	unavailable_to_build = 1
+
+/datum/robolimb/cybersolutions
```
Comment (Anewbe) [inline code/modules/organs/robolimbs.dm]: This isn't going to work. You need to have it be a different path than any of the other datums.

Comment (Belsima) [reply code/modules/organs/robolimbs.dm]: Ah, thanks.

File: code/datums/supplypacks/robotics.dm (hunk 7 above)
Comment (Belsima) [reply code/datums/supplypacks/robotics.dm]: I'll make it a consistent *0.8 rounded up from the company they're a knockoff from.

File: code/datums/supplypacks/robotics.dm (hunk 1 above)
Comment (Belsima) [reply code/datums/supplypacks/robotics.dm]: These are just equal to Morpheus given their consistently low quality.

**Comment by Wutnold124** [comment]
all of these sprites are great and i want to hug whoever made 'em

**Comment by Anewbe** [comment]
@Belsima This was discussed at the staff meeting, we  like the sprites, but we plan to implement them differently. If you would mind terribly adding the sprites, but not the code, we would appreciate it. If you would like to take it a step further and do the actual implementation, let me know and I'll try to lay it out for you.

**Comment by Belsima** [comment]
1. What would that implementation be?
And 2. Would implementing the Cyber Solutions brand on its own be fine, given it doesn't match the look of the rest given?

**Comment by Anewbe** [comment]
1.) Alternate models for existing TSC brands.

2.) Possibly, I'd rather we have lore for them before we implement them, though.

**Comment by Belsima** [comment]
We actually do have a single line of lore for Cyber Solutions in http://ss13polaris.com/wiki/doku.php?id=lore:drone which just says they're cheap.

**Comment by Anewbe** [comment]
Good find.

**Comment by Belsima** [comment]
Implemented as alternatives like the current alternative heads are.

**Comment by Anewbe** [comment]
Oh, right, names will be discussed at a later date, forgot about that.

**Comment by Belsima** [comment]
...ah. Alright. I just went ahead with what I thought was fitting and asked elg for a couple.

**Comment by Anewbe** [comment]
Lore staff seems to be fine with the names:
1.) Hephaestus Athena
2.) Xion Hull
3.) Bishop Rook
4.) Ward-Takahashi Spirit

**Comment by Belsima** [comment]
Committed the new names.

**Comment by Anewbe** [comment]
This was improperly credited, according to my sources credit goes to Lohikar.

**Pull request merged**